docs/changelog.md

```diff
@@ -3,6 +3,9 @@ id: changelog
 title: Changelog
 sidebar_label: Changelog
 ---
+## v2.3.0
+
+- Added [`FOREACH`](/cypher-manual/extensions.md) clause. [#351](https://github.com/memgraph/memgraph/pull/351)
```

Comment:
@g-despot Can you plz explain to me why the above link is brokern? Maybe I am referencing it in the wrong way?

Reply:
My mistake, I just started reviewing the PR and noticed that you used an id for the file that's not the same as the file name. I will change it from `extension-clauses` to `extension`. 🙂

Reply:
```suggestion
- Added [`FOREACH`](/cypher-manual/extension-clauses) clause. [#351](https://github.com/memgraph/memgraph/pull/351)
```

This works, based on the notion page I guess this is a [link from a 1st level directory to another 1st level directory](https://www.notion.so/memgraph/Memgraph-Docs-059e332bce37439f86ab979ca77ad5ce#0d77e3dfd7e44feabf0ae8cfb7cef839) and that's why it must use the title instead of the file name.

Reply:
And of course Benjo is right, I missed the fact that we are in the `/docs` directory which means that we can't use relative file paths like `/cypher-manual/extensions.md`. Instead, slugs are used when linking between directories: `/cypher-manual/extensions` (if the file were named `extensions.md` and the id was `extensions`).